### ISSUE TITLE
Templates are useless

### ISSUE THREAD

**Issue by rmaddox** [sendwithus/templates#26]
While I do understand how you can use them, my opinion is that these are useless templates for reason that you are using tables and content is in various <td> tags, which is basically against the HTML guidelines. This makes it so hard to simply provide content, and run it in the template. It is HTML template, but not usable.

**Comment by demoore** [comment]
Hey @rmaddox can you be a little more specific about _which_ parts are against _which_ guidelines. 

Thanks!

**Comment by rmaddox** [comment]
I get the feeling what you get now. I am not a new user of templates, and I am Internet user since before the browser came. I was among first in my area to open up new websites. And I am using now over 150 various templates. Still searching for simplest one, I was thinking your is simplest, but it is not. It is full of tables inside, <td> etc.

When we speak of a template, I think of: **content to be put in the template**, isn't it?

This is here what I think that is content:

```
                        <h1>Something Big...</h1>

                        <h2>Hi Stranger,</h2>

                        <p>Kielbasa venison ball tip shankle. Boudin prosciutto landjaeger, pancetta jowl turkey tri-tip porchetta beef pork loin drumstick. Frankfurter short ribs kevin pig ribeye drumstick bacon kielbasa. Pork loin brisket biltong, pork belly filet mignon ribeye pig ground round porchetta turducken turkey. Pork belly beef ribs sausage ham hock, ham doner frankfurter pork chop tail meatball beef pig meatloaf short ribs shoulder. Filet mignon ham hock kielbasa beef ribs shank. Venison swine beef ribs sausage pastrami shoulder.</p>

                        <p>By the way, if you're wondering where you can find more of this fine meaty filler, visit <a href="http://baconipsum.com">Bacon Ipsum</a>.</p>

                        <p><em>– Mr. Pen</em></p>

```

And here is what you think how I shall insert such content:

```
<table class="body-wrap">
    <tr>
        <td class="container">

            <!-- Message start -->
            <table>
                <tr>
                    <td align="center" class="masthead">

                        <h1>Something Big...</h1>

                    </td>
                </tr>
                <tr>
                    <td class="content">

                        <h2>Hi Stranger,</h2>

                        <p>Kielbasa venison ball tip shankle. Boudin prosciutto landjaeger, pancetta jowl turkey tri-tip porchetta beef pork loin drumstick. Frankfurter short ribs kevin pig ribeye drumstick bacon kielbasa. Pork loin brisket biltong, pork belly filet mignon ribeye pig ground round porchetta turducken turkey. Pork belly beef ribs sausage ham hock, ham doner frankfurter pork chop tail meatball beef pig meatloaf short ribs shoulder. Filet mignon ham hock kielbasa beef ribs shank. Venison swine beef ribs sausage pastrami shoulder.</p>

                        <table>
                            <tr>
                                <td align="center">
                                    <p>
                                        <a href="#" class="button">Share the Awesomeness</a>
                                    </p>
                                </td>
                            </tr>
                        </table>

                        <p>By the way, if you're wondering where you can find more of this fine meaty filler, visit <a href="http://baconipsum.com">Bacon Ipsum</a>.</p>

                        <p><em>– Mr. Pen</em></p>

                    </td>
                </tr>
            </table>

        </td>
    </tr>
    <tr>
        <td class="container">

            <!-- Message start -->
            <table>
                <tr>
                    <td class="content footer" align="center">
                        <p>Sent by <a href="#">Company Name</a>, 1234 Yellow Brick Road, OZ, 99999</p>
                        <p><a href="mailto:">hello@company.com</a> | <a href="#">Unsubscribe</a></p>
                    </td>
                </tr>
            </table>

        </td>
    </tr>
</table>
```

- if I wish to place the button, I am supposed to make a table?!
- for footer I understand, it is maybe same for all content
- content is in <td>
- header is also in other <td>

So how am I supposed to insert header and content? Do you get the idea? I know how to do it, I have made my own CMS several times. I know how to insert variables, but it is just not user friendly.

If there is anything like <content>here is content</content> then I expect  to put everything there, and not to separate header in other place outside of content scope.

Basically, template should be template, separate from content, so that any content may be quickly changed and replaced. I would simply insert a variable inside like {$page_content} or something and it would appear as HTML in the template. 

Sorry for disturbance.

**Comment by demoore** [comment]
Tables are annoying and a gross way to do layout — I know. The issue is email HTML is stuck in the year 2001, back when the first browser war had reached it's peak and `<blink>` and `<marquee>` tags roamed the web freely. We're still feeling the fallout today, check out [Campaign Monitor's chart](https://www.campaignmonitor.com/css/) for a list of which email clients support which CSS rules. It's crazy.

These email templates are written in such a way to maximize compatibility and responsiveness across most email clients. It makes it more difficult to template, but you save yourself some debugging time when a user on Outlook 2007 complains about a button not rendering properly.

**Comment by rmaddox** [comment]
yes, thank you. 

I was thinking yours was the simplest, but have to degrade it down. It looks nice for me. I just removed the tables and all. I need blank
